src/rouge/core/workflow/pipeline.py

```diff
@@ -68,15 +69,22 @@ def run(self, issue_id: int, adw_id: str, logger: Logger) -> bool:
 
 
 def get_default_pipeline() -> List[WorkflowStep]:
-    """Create the default workflow pipeline with all 11 steps.
+    """Create the default workflow pipeline.
+
+    The pipeline conditionally includes a PR/MR creation step based on the
+    DEV_SEC_OPS_PLATFORM environment variable:
+    - "github": includes CreateGitHubPullRequestStep
+    - "gitlab": includes CreateGitLabPullRequestStep
+    - unset or other value: no PR/MR step included
```

Comment:
The new env var `DEV_SEC_OPS_PLATFORM` should be added to `.env.example` and noted in README.md as well.